static/styles/portico/landing_page.css

```diff
@@ -1573,6 +1573,20 @@ nav {
     }
 }
 
+.carousel-control.left:hover #prev {
+    display: block;
+}
+
+.carousel-control.right:hover #next {
```

Comment:
Also you don't need `.carousel-control.left:hover #prev` and `.carousel-control.right:hover #next`,
you can just instead,
```css
  &:hover { /* Line 1563 */
+       display: block;
        color: hsl(220, 23%, 33%);
    }
```

Reply:
Is this what you meant..? It isn't working for me locally. 

``` css
.tour .carousel-control {  /* Line 1553 */
    background: 0;
    border: 0;
    color: hsl(223, 13%, 52%);
    font-size: 2em;
    top: 50%;
    font-weight: 400;
    opacity: 1;
    border-radius: 0;
    
+  .carousel-label {
+       display: none;
+       font-family: inherit;
+       font-size: 14px;
+   }
     
    &.right {
        right: 40px;
    }
    
    &.left {
        left: 40px;
    }
    
    &:hover {
+       display: block;
        color: hsl(220, 23%, 33%);

    }

}
```

Reply:
Ohh! My bad try this,
```css
.tour .carousel-control {  /* Line 1553 */
    background: 0;
    border: 0;
    color: hsl(223, 13%, 52%);
    font-size: 2em;
    top: 50%;
    font-weight: 400;
    opacity: 1;
    border-radius: 0;
    
+  .carousel-label {
+       display: none;
+       font-family: inherit;
+       font-size: 14px;
+   }
     
    &.right {
        right: 40px;
    }
    
    &.left {
        left: 40px;
    }
    
    &:hover {
        color: hsl(220, 23%, 33%);
+        .carousel-label {
+        display: block;
+        }
    }

}
```

Reply:
Yep works! Thanks